### PR TITLE
updated Ryan Cooper bio

### PR DESCRIPTION
**Pull request by cooperrc**
I updated my bio with links to my UConn website and some other [github.uconn.edu](https://github.uconn.edu) resources available.

### CODE REVIEW

File: participants.md [hunk 1]
```diff
@@ -21,10 +21,18 @@ His materials are also available [on Github](https://github.com/bbadzioch/MTH309
...
-[Ryan](https://me.engr.uconn.edu/blog/faculty/cooper-ryan/) is assistant professor-in-residence at the University of Connecticut Department of Mechanical Engineering. 
-He is now teaching computational mechanics using Jupyter, and his course materials are available [on GitHub](https://github.uconn.edu/rcc02007/Computational_Mechanics).
+### Ryan C. Cooper
+
+[Ryan](https://ryan-c-cooper.uconn.edu) is assistant professor-in-residence at
+the University of Connecticut Department of Mechanical Engineering. He aims to
+stengthen problem-solving skills with project-based learning and numerical
+methods.  He has used Jupyter to enhance learning in Dynamics, Vibrations,
+upper-level undergraduate laboratories, and is now teaching computational
+mechanics using Jupyter. His course materials are available [on GitHub
```
Comment (bryanwweber) [inline participants.md:31]: This link looks a little weird to me, did you mean `on GitHub: [Computational ...`?

Comment (kyleniemeyer) [reply participants.md:31]: `[on GitHub](https://github.uconn.edu/rcc02007/Computational_Mechanics)` should appear as [on GitHub](https://github.uconn.edu/rcc02007/Computational_Mechanics), which seems fine to me

Comment (bryanwweber) [reply participants.md:31]: I think it says `[on GitHub Computational Mechanics](...)` if I'm reading the Markdown correctly

Comment (cooperrc) [reply participants.md:31]: I had made the link 

"[on Github Computational Mechanics](https://github.uconn.edu/rcc02007/Computational_Mechanics)", 

but its fine to change it to:

"on Github: [Computational Mechanics](https://github.uconn.edu/rcc02007/Computational_Mechanics)"

I just like __big links__

**Comment by bryanwweber** [comment]
@cooperrc Take a look at the updated format and let me know if it makes sense to you

**Comment by cooperrc** [comment]
Looks great, thanks @bryanwweber

**Pull request merged**